README.rst

```diff
@@ -218,6 +233,13 @@ template ``context_processors`` in ``settings.py`` as shown below.
 Settings
 ^^^^^^^^
 
+``OPENWISP_ADMIN_SITE_CLASS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**default**: ``openwisp_utils.admin_theme.admin.OpenwispAdminSite``
+
+Path to custom admin site class
```

Comment:
Change to:

`If you need to use a customized admin site class, you can use this setting.`